### PR TITLE
Update Slacker to 0.6.2 and add username and icon

### PR DESCRIPTION
**Pull request by abtris**
Added new version Slacker support more options.

https://github.com/os/slacker/blob/master/slacker/__init__.py#L227

### CODE REVIEW

File: pavement.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 NAME = 'blackbelt'
 VERSION = '0.12.1'
 
-requires = ['click', 'requests', 'PyGithub', 'slacker', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
+requires = ['click', 'requests', 'PyGithub', 'slacker==0.6.2', 'pyopenssl', 'ndg-httpsclient', 'pyasn1']
```
Comment (abtris) [inline pavement.py]: @Almad I'm not sure if this is ok or we can add some minimal version to `requirements.txt`

Comment (Almad) [inline pavement.py]: @abtris It should be on both places. 

I _think_ @honzajavorek told me that we actually can get rid of `requirements.txt`, so ask him.

File: blackbelt/slack.py [hunk 2]
```diff
@@ -12,10 +12,14 @@ def __init__(self, token=None):
 
         self.token = token
 
+    def get_user_id(self):
+        return self.slack.auth.test().body['user_id']
+
     def post_message(self, message, room):
-        return self.slack.chat.post_message(room, message)
+        return self.slack.chat.post_message(room, message, username = "Black Belt", icon_emoji = ":black_joker:")
```
Comment (Almad) [inline blackbelt/slack.py]: I think we want to have a logo at some point :P

Comment (abtris) [inline blackbelt/slack.py]: We don't have logo. We can ping @natalievenuto or @jandvorak if they create some for Black Belt.

File: pavement.py (hunk 1 above)
Comment (honzajavorek) [inline pavement.py]: The idea is described here: https://caremad.io/2013/07/setup-vs-requirement/. Basically, `setup.py` is similar to `package.json`, `requirements.txt` is similar to shrinkwrap.

BB isn't deployable project, so in ideal world it should have only `setup.py` with as loosely pinned dependencies as possible.

And then there is Paver. I think it's something like Grunt, basically a task runner. Most of Python projects don't use it for managing dependencies. Specifying requirements in `pavement.py` is Paver-specific and I have no idea what to do with it.

What we should strive for is to have at least a single source of truth, single place where to put new dependencies.

Comment (abtris) [inline pavement.py]: For now, I add minimal version `Slacker` to both places and in some future PR we can solve this problem.

Comment (honzajavorek) [inline pavement.py]: :+1:

**Comment by abtris** [comment]
@honzajavorek Ready to merge?

**Comment by Almad** [comment]
re: paver, syntax there is the same as for `setup.py`, it just proxies the methods.

Might be worth separating setup and paver tasks if people are confused by it &shrug;

File: blackbelt/slack.py [hunk 3]
```diff
@@ -12,10 +12,14 @@ def __init__(self, token=None):
 
         self.token = token
 
+    def get_user_id(self):
+        return self.slack.auth.test().body['user_id']
```
Comment (honzajavorek) [inline blackbelt/slack.py]: what does the `test()` do? is it a name of something?

**Comment by honzajavorek** [comment]
Except of wondering what the `test()` is about it looks good :+1:

File: blackbelt/slack.py (hunk 3 above)
Comment (abtris) [inline blackbelt/slack.py]: It's method from API who returns all data about authorized user.

Comment (honzajavorek) [inline blackbelt/slack.py]: Hah, very strange name...

**Pull request merged**